test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -111,18 +112,12 @@ def test_jsonapi_parser_registered
     end
 
     def test_jsonapi_renderer_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
+      expected = { 'name' => 'Johnny Rico' }
+
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal expected, JSON.parse(response.body)['data']['attributes']
```

Comment:
i think so. But I can't get your code to work.  The dependency required is getting a bit too much.  For now, i have updated expected object to include all relationships so that we check for exact output.